test/integration/narrow/routes/healthz.test.js

```diff
@@ -1,10 +1,14 @@
+jest.mock('../../../../app/auth', () => ({
+  getKeys: jest.fn().mockResolvedValue({ publicKey: 'publicKey' }),
+  validateToken: jest.fn().mockResolvedValue({ isValid: true })
+}))
 const { createServer } = require('../../../../app/server')
 
 let server
 
 beforeEach(async () => {
   server = await createServer()
-  await server.start()
+  await server.initialize()
```

Comment:
Whats the difference between `start` and `initialize`?

Reply:
For the most part they do the same thing.  But start will bind the server to an actual port, whereas initialize doesn't.  So for tests we don't need to actually do that and it does risk collisions in CI.  Albeit our use of containers pretty much nullifies that risk.